Dockerfile.template

```
@@ -165,7 +165,7 @@ RUN set -eux; \
 	apt-get install -y --no-install-recommends \
 # add tzdata explicitly for https://github.com/docker-library/valkey/issues/138 (see also https://bugs.debian.org/837060 and related)
 		tzdata \
-		libssl3 \
+		{{ if .debian.version == "trixie" then "libssl3t64" else "libssl3" end }} \
```

```suggestion
		{{ if .debian.version != "bookworm" then "libssl3t64" else "libssl3" end }} \
```
expecting all newer use the `libssl3t64`
WDTY?